README.md

```diff
@@ -40,6 +40,7 @@ This plugin queries OPA providing the following properties as `input`
 - `headers` the Fastify headers object
 - `parent` the Mercurius parent object of the field/object which got queried
 - `args` the Mercurius args object of the field/object which got queried
+- `options` static untyped properties defined in the directive arguments _(optional)_
```

Comment:
Maybe we can call it `input` li OPA does?

Reply:
Passing an `input` field into the policy input means that it would be accessed in rego as `input.input`, I'd much prefer `input.options`